rules/react/walmart.js

```diff
@@ -11,12 +11,22 @@ module.exports = {
   "rules": {
     // Prevent missing displayName in a React component definition
     "react/display-name": 0,
+    // Forbid certain propTypes
+    "react/forbid-prop-types": 0,
     // Enforce boolean attributes notation in JSX
     "react/jsx-boolean-value": 2,
+    // Validate closing bracket location in JSX
+    "react/jsx-closing-bracket-location": 0,
```

Comment:
It is a style thing, but I vote for turning this on, unless it conflicts with already-established patterns.